### ISSUE TITLE
[HW] Pamela Jung

### ISSUE THREAD

**Issue by pamify** [datastaxdevs/appdev-week1-todolist#40]
**Name:** Pamela Jung
**Email:** pamjung@gmail.com
**Linkedin Profile:** https://linkedin.com/in/pamjung
<img width="1512" alt="ToDoList - React Basics" src="https://user-images.githubusercontent.com/72463342/124947598-f79d9c00-dfdd-11eb-9a4a-9fbd2bd1dfce.png">
<img width="1289" alt="ToDoList - REST todos" src="https://user-images.githubusercontent.com/72463342/124947614-fb312300-dfdd-11eb-896e-512e245dcbd0.png">

Attach the homework screenshots below:
-----------------------------------------

<SCREENSHOTS>

**Comment by SonicDMG** [comment]
Great job Pamela and you are on your way to the gold badge.

**Comment by SonicDMG** [comment]
Here's your badge -> https://api.badgr.io/public/assertions/y1zOCn9_Tg-tlPrxY-ifkA?identity__email=pamjung%40gmail.com

**Comment by pamify** [comment]
Hi David, 

I was unintentionally awarded for the Tiktok badge, but I completed the ToDoList workshop. 

I’m signed up for TikTok workshop on Wednesday.

Thanks,
Pamela

**Comment by SonicDMG** [comment]
You're correct. I just noticed that myself. Sorry about that. Here's the correct badge -> https://api.badgr.io/public/assertions/cItNyIHeRLWcEqIAiukKWA?identity__email=nidhiskr2014%40gmail.com

**Comment by pamify** [comment]
I got it.  Thanks for the quick reply.

I have an unrelated question hoping you can help.  My friend and I attended
the Netflix Clone workshop live and there was a Giveaway Claim Form.  Is
the giveaway something she can claim at a future workshop?  She is
interested in the voucher for the Cassandra certification.

Best,
Pamela

**Comment by pamify** [comment]
I received the certification voucher.  Is it possible to have another
voucher for my friend, Kathy?

Thank you.

**Comment by SonicDMG** [comment]
Yes, she can claim a giveaway at a future workshop. Also, if she attended a previous one have her email jack.fryer@datastax.com. If she registered and attended with the same email we should be able to help her out.

**Comment by pamify** [comment]
Thanks.  I really appreciate it.  See you on Wednesday!
